scripts/build-engine/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cocos/build-engine",
-  "version": "4.3.17",
+  "version": "4.3.17-oh.1",
```

Comment:
Are you sure to change this file like this?

Reply:
@PPpro

Reply:
yes, we need to change the current published version, so that next time we publish a new version from this.

Reply:
Ok